src/pages/ShopingCart.page.js

```diff
@@ -22,4 +36,28 @@ export class ShopingCartPage extends BaseSwagLabPage {
     async removeCartItemById(id) {
         await this.cartItems.nth(id).locator(this.removeItemSelector).click();
     }
+
+    async clickCheckout() {
+        await this.page.locator(this.checkoutSelector).click();
+    }
+
+    async cartItemData(id) {
```

Comment:
Let's think about the way to combine or reuse the code in this page and in Inventory.page.js file